activity.js

```diff
@@ -0,0 +1,22 @@
+class Activity {
```

Comment:
Looks great. Might change the parameter names in the future depending on what we find the most helpful for us.